src/vector/feature.rs

```diff
@@ -90,7 +90,7 @@ impl<'a> Feature<'a> {
         match field_type {
             OGRFieldType::OFTString => {
                 let rv = unsafe { gdal_sys::OGR_F_GetFieldAsString(self.c_feature, field_idx) };
```

Comment:
`rv` is owned by `c_feature` so we don't need to free it?

Reply:
Yes:

> This string is internal, and should not be modified, or freed. Its lifetime may be very brief.